wallpaper-bindings/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "wallpaper-bindings"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+more-wallpapers = {version = "0.3.1", features = ["fallback"]}
+
+[lib]
+name = "wallpaper-bindings"
```

Comment:
also this has to be `wallpaper_bindings` or it won't compile. Rust doesn't allow hyphens for some reason (I think something to do with URLs and crates.io).

Reply:
oh weird lmao 1s